controllers/scripts/create_pod_status_patch.py

```diff
@@ -374,132 +380,150 @@ def init_volumes(pod_name, config):
 
-            elif volume.effective_init_method == "blkdiscard":
+                    dd = 'dd if=/dev/zero of={volume_path} bs=1M 2> /tmp/init-stderr || grep -q "No space left on device" ' \
+                        '/tmp/init-stderr'.format(
+                        volume_path=volume.get_mount_point())
+                    futures[executor.submit(lambda: execute(cmd=dd))] = dd
+                    logging.info(f"{volume} - Submitted")
 
-                blkdiskard = "blkdiscard {volume_path}".format(
-                    volume_path=volume.get_mount_point())
-                execute(blkdiskard)
-                logging.info(f"{volume} - Initialized")
+                elif volume.effective_init_method == "blkdiscard":
 
-            elif volume.effective_init_method == "none":
-                logging.info(f"{volume} - Pass through")
-            else:
-                logging.error(f"{volume} - Has invalid effective method")
-                raise ValueError(f"{volume} - Has invalid effective method")
+                    blkdiskard = "blkdiscard {volume_path}".format(
+                        volume_path=volume.get_mount_point())
+                    futures[executor.submit(lambda: execute(cmd=blkdiskard))] = blkdiskard
+                    logging.info(f"{volume} - Submitted")
 
-        elif volume.volume_mode == "Filesystem":
-            logging.debug(f"In Filesystem initialization: {volume}")
-            if not os.path.exists(volume.get_mount_point()):
-                logging.error(f"pod-name: {pod_name} volume-name: {volume.volume_name} - Mounting point "
-                              f"does not exists")
-                raise FileNotFoundError(f"{volume} Volume path not found")
+                elif volume.effective_init_method == "none":
+                    logging.info(f"{volume} - Pass through")
+                else:
+                    logging.error(f"{volume} - Has invalid effective method")
+                    raise ValueError(f"{volume} - Has invalid effective method")
+
+            elif volume.volume_mode == "Filesystem":
+                logging.debug(f"In Filesystem initialization: {volume}")
+                if not os.path.exists(volume.get_mount_point()):
+                    logging.error(f"pod-name: {pod_name} volume-name: {volume.volume_name} - Mounting point "
+                                  f"does not exists")
+                    raise FileNotFoundError(f"{volume} Volume path not found")
 
-            if volume.effective_init_method == "deleteFiles":
+                if volume.effective_init_method == "deleteFiles":
 
-                find = "find {volume_path} -type f -delete".format(
-                    volume_path=volume.get_mount_point())
-                execute(find)
-                logging.info(f"{volume} - Initialized")
+                    find = "find {volume_path} -type f -delete".format(
+                        volume_path=volume.get_mount_point())
+                    execute(find)
```

Comment:
Here, we are running the command directly but this is being queued in the wipe function.

Reply:
Should we just queue commands for the devices and run directly for files?